keywords/couchbaseserver.py

```diff
@@ -219,10 +220,12 @@ def wait_for_ready_state(self):
             # All nodes are heathy if it made it to here
             break
 
-    def _create_internal_rbac_bucket_user(self, bucketname):
+    def _create_internal_rbac_bucket_user(self, bucketname, cluster_config):
```

Comment:
This method is used by conftest file in CBL tester folder which is not modified according to this change. Cluster_config is not passed in conftest file , tests under CBL tests which are functional tests and topology specific tests will fail

Reply:
added into conftest file.